src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+
+@import "variables";
+
+body {
+  margin: 0;
+}
+
+.clock {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+  @include w-h(500px,500px);
+}
+
+.clock__center {
+  @include w-h(25px,25px);
+  background-color: #4b9aaa;
+  border-radius: 50%;
+  margin: auto;
+}
+
```

Comment:
I would add a z-index so that the minute hand is above the second and the center is above the two hands, as usual in a clock